SchoolHousing/SchoolHousing/Pages/Index.cshtml.cs

```diff
@@ -10,17 +10,25 @@
 using ElementarySchoolResponse;
 public class IndexModel : PageModel
 {
-    public void OnGet()
+    public JsonResult OnGet()
     {
-        using (var webClient = new WebClient())
-        {
-            string schoolString = webClient.DownloadString("https://data.cityofchicago.org/resource/tj8h-mnuv.json");
+            string schoolString = getJsonString("https://data.cityofchicago.org/resource/tj8h-mnuv.json");
             var schoolData = ElementarySchool.FromJson(schoolString);
-            ViewData["ElementarySchool"] = schoolData;
```

Comment:
Removed redundant code push to View

Reply:
Ahhhh.... how are you pushing to view without ViewData[]?  Or is this a side effect of changing from HTML page to JSON output?  Either way, this is a functionality change again, which the group might not want.